src/test/java/com/logigear/tranining/user/management/TC_004_LoginPage_VerifyLoginSuccessfullyWithDifferentRepository.java

```diff
@@ -0,0 +1,27 @@
+package com.logigear.tranining.user.management;
+
+import com.logigear.training.common.GlobalVariables;
+import com.logigear.training.common.TestBase;
+import com.logigear.training.page.HomePage;
+import com.logigear.training.page.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TC_004_LoginPage_VerifyLoginSuccessfullyWithDifferentRepository extends TestBase {
+    @Test
+    public void TC_001() {
+        LoginPage loginPage = new LoginPage();
+
+        //Pre-condition: Login to SampleRepository
```

Comment:
I think you should create a method "public void PreCondition()" with precondition